development-defaults.ini

```diff
@@ -1,5 +1,5 @@
 
-path = "/uber"
```

Comment:
Hah, what do you know, it IS puppetized. https://github.com/magfest/ubersystem-puppet/blob/master/templates/uber-development.ini.erb#L4 and the default in our puppet is 'uber'.

While I personally would prefer MAGFest's system to also use the 'rams' prefix, I suspect that people who have memorized URLs will hang me if I do that. But yeah, this shouldn't cause a problem for us if we merge it.